modules/hcp-aws-tags.adoc

```diff
@@ -0,0 +1,80 @@
+//Module included in the following assemblies:
+// hosted_control_planes/hcp-manage/hcp-manage-aws.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-aws-tags_{context}"]
+= Adding or updating {aws-short} tags for a hosted cluster
+
+As a hosted cluster administrator, you can add or update {aws-first} tags without needing to re-create your hosted cluster. _Tags_ are key-value pairs that are attached to {aws-short} resources for management and automation.
+
+You might want to use tags for the following purposes:
+
+* Managing access controls.
+* Tracking chargeback or showback.
+* Managing cloud IAM conditional permissions.
+* Aggregating resources based on tags. For example, you can query tags to calculate resource usage and billing costs.
+
+You can enable tags to be updated for several different types of resources, including EFS access points, load balancer resources, Amazon EBS volumes, IAM users, and {aws-short} S3.
+
+[IMPORTANT]
+====
+On network load balancers, tags cannot be updated. The {aws-short} load balancer reconciles whatever tags are in the `HostedCluster` resource. If you try to add new tag, the load balancer overwrites everything.
```

Comment:
Everything seems a bit strong to me. Not sure what other wording to use. 
```suggestion
On network load balancers, tags cannot be updated. The {aws-short} load balancer reconciles whatever tags are in the `HostedCluster` resource. If you try to a add new tag, the load balancer overwrites the tag.
```

Reply:
Fixed